builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -281,17 +306,33 @@ def deletePolicies(serviceId, authToken, aclUrl) {
 }
 
 def cleanupEventSourceMapping(env) {
-  def lambda_arn = "arn:aws:lambda:${configLoader.AWS.REGION}:${configLoader.AWS.ACCOUNTID}:function:${configLoader.INSTANCE_PREFIX}-${service_config['domain']}-${service_config['service']}-${env}"
-  def assets_api = g_base_url + "/jazz/assets"
-  lambdaEvents.deleteEventSourceMapping(lambda_arn, assets_api, auth_token,service_config, env)
+		withCredentials([
+		[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: service_config.credentialId, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+	]) {
+		try {
+			def randomString = utilModule.generateRequestId();
+			def credsId = "jazz-${randomString}";
+			sh "aws configure set profile.${credsId}.region ${service_config.region}"
+			sh "aws configure set profile.${credsId}.aws_access_key_id $AWS_ACCESS_KEY_ID"
+			sh "aws configure set profile.${credsId}.aws_secret_access_key $AWS_SECRET_ACCESS_KEY"
+
+  		def lambda_arn = "arn:aws:lambda:${service_config['region']}:${service_config.accountId}:function:${configLoader.INSTANCE_PREFIX}-${service_config['domain']}-${service_config['service']}-${env}"
+  		def assets_api = g_base_url + "/jazz/assets"
+  		lambdaEvents.deleteEventSourceMapping(lambda_arn, assets_api, auth_token,service_config, env,credsId)
+
+			resetCredentials(credsId)
```

Comment:
This has to be in finally. Reset credentials even when there is an exception.

Let's make sure that this is covered across all other JenkinsFiles.

Reply:
done